src/onediff/infer_compiler/backends/oneflow/deployable_module.py

```
@@ -247,6 +247,25 @@ def apply_online_quant(self, quant_config):
             >>> model.apply_online_quant(quant_config)
         """
         self._deployable_module_quant_config = quant_config
+    def __init__(self, torch_module, oneflow_module, dynamic=True, options=None):
```

Ruff

Redefinition of unused `__init__` from line 53

(F811)